filters/badware.txt

```diff
@@ -1240,3 +1240,8 @@ okashimag.com,tech4yougadgets.com##+js(aopr, Notification)
 ||jinshanduba.org.cn^$all
 ! https://bbs.kafan.cn/thread-2170747-1-1.html – malicious script
 ||phpstat.cntcm.com.cn/phpstat/count/abceffgh/abceffgh.js^$script
+
+! https://github.com/uBlockOrigin/uAssets/pull/11041
+
```

Comment:
```suggestion

```
Why is there a blank line here?

Reply:
Also might want to add `https://cdn.discordapp.com/attachments/916391647955279943/922200309856612362/HypixelDupe_1.0.8.jar` as that is the download url

Reply:
> https://cdn.discordapp.com/attachments/916391647955279943/922200309856612362/HypixelDupe_1.0.8.jar

Added the entire discord server that this malware is on